it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationControllerWithAssuranceFlagISpec.scala

```diff
@@ -40,16 +40,16 @@ class CheckAgencyControllerWithAssuranceFlagISpec extends CheckAgencyControllerI
 
       implicit val request = authenticatedAs(subscribingAgentEnrolledForNonMTD)
         .withFormUrlEncodedBody("utr" -> validUtr.value, "postcode" -> validPostcode)
-      val result = await(controller.checkAgencyStatus(Some(CheckAgencyController.validBusinessTypes.head))(request))
+      val result = await(controller.submitBusinessDetailsForm(Some(BusinessIdentificationController.validBusinessTypes.head))(request))
 
       status(result) shouldBe 303
-      redirectLocation(result) shouldBe Some(routes.CheckAgencyController.showConfirmYourAgency().url)
+      redirectLocation(result) shouldBe Some(routes.BusinessIdentificationController.showConfirmBusinessForm().url)
       sessionStoreService.currentSession.knownFactsResult shouldBe Some(
         KnownFactsResult(validUtr, validPostcode, "My Agency", isSubscribedToAgentServices = false, Some(businessAddress), Some("someone@example.com")))
       verifyAgentAssuranceAuditRequestSent(
         passPayeAgentAssuranceCheck = Some(true),
         passSaAgentAssuranceCheck = Some(true))
-      metricShouldExistAndBeUpdated("Count-Subscription-CheckAgency-Success")
+      metricShouldExistAndBeUpdated("Count-Subscription-ConfirmBusiness-Success")
```

Comment:
A couple of metric names have changed. I think this would affect the statistics on the dashboard, so we might want to either keep them as is, or consider updating the dashboard (but it'd lose the old statistics):
"Count-Subscription-ConfirmBusiness-Success" changed to "Count-Subscription-CheckAgency-Success"
"Count-Subscription-BusinessDetails-Start" changed to "Count-Subscription-CheckAgency-Start"